### PR TITLE
fix: icons formatting

### LINKED ISSUE #1342: Fix logger formatting
# Description

The logger icons are not correctly formatted and need to be updated.

# Environment
OS: `Ubuntu 22.04 LTS`
Version: `v2.3.7`
 
# Steps To Reproduce

`kubescape scan resource.yaml`

# Expected behavior

The icons should be well formatted.

# Actual Behavior

Icons are of different width and not really well formatted.

# Additional context

This will require changes to the `go-logger` library.

### PR DESCRIPTION
**Pull request by XDRAGON2002**
## Overview

As mentioned in the issue, this PR fixes the formatting of the icons.

## Additional Information

To be able to be merged, it'd require https://github.com/kubescape/go-logger/pull/16 to be merged first, a new release for `go-logger` to be generated and then the version to be updated in `go.mod` here.

## How to Test

`kubescape scan resource.yaml`

## Examples/Screenshots

Before:

image

After:

image

## Related issues/PRs:

Resolved #1342 

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by XDRAGON2002** [comment]
@craigbox @dwertent this solves both of your requirements, restoring the previous `prettylogger` and adding a new logger instead as well as ensuring the correct formatting.

**Comment by XDRAGON2002** [comment]
https://github.com/kubescape/go-logger/pull/16 was merged, bumping the version.

File: core/core/scan.go [hunk 1]
```diff
@@ -125,6 +125,7 @@ func GetOutputPrinters(scanInfo *cautils.ScanInfo, ctx context.Context) []printe
 
 func (ks *Kubescape) Scan(ctx context.Context, scanInfo *cautils.ScanInfo) (*resultshandling.ResultsHandler, error) {
 	ctxInit, spanInit := otel.Tracer("").Start(ctx, "initialization")
+	logger.InitLogger("icon")
```
Comment (dwertent) [inline core/core/scan.go:128]: Use the logger name from the const in the package

**Review by dwertent — CHANGES_REQUESTED** [review]

**Comment by dwertent** [comment]
@Daniel-GrunbergerCA This will fix the indentation issue we spoke about :)

File: core/core/scan.go (hunk 1 above)
Comment (XDRAGON2002) [reply core/core/scan.go:128]: @dwertent are you referring to importing the entire `iconlogger` package to get that? Or to get it from `ListLoggerNames()[1]`?

Comment (dwertent) [reply core/core/scan.go:128]: Import the `iconlogger` pkg so you can use the logger name

Comment (XDRAGON2002) [reply core/core/scan.go]: @dwertent made the change! Thanks!

**Review by dwertent — APPROVED** [review]

**Pull request merged**